From 123e3211fe95c5a9b439055133f9a66367f159e4 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 14:12:20 -0400
Subject: [ethernet] igb: Replace 1/0 return values with true/false

Message-id: <1409062359-13063-63-git-send-email-sassmann@redhat.com>
Patchwork-id: 89184
O-Subject: [RHEL7.1 PATCH 62/81] igb: Replace 1/0 return values with true/false
Bugzilla: 1091121
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Carolyn Wyborny <carolyn.wyborny@intel.com>

This patch fixes issues found by updated coccicheck.

Signed-off-by: Carolyn Wyborny <carolyn.wyborny@intel.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit f6f38e270e9be53900d230c9a1eef3b1b5891f03)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igb/igb_ethtool.c b/drivers/net/ethernet/intel/igb/igb_ethtool.c
index 8fe6e43..9780bc3 100644
--- a/drivers/net/ethernet/intel/igb/igb_ethtool.c
+++ b/drivers/net/ethernet/intel/igb/igb_ethtool.c
@@ -1208,11 +1208,11 @@ static bool reg_pattern_test(struct igb_adapter *adapter, u64 *data,
     "pattern test reg %04X failed: got 0x%08X expected 0x%08X\n",
     reg, val, (_test[pat] & write & mask));
    *data = reg;
-   return 1;
+   return true;
   }
  }
 
- return 0;
+ return false;
 }
 
 static bool reg_set_and_check(struct igb_adapter *adapter, u64 *data,
@@ -1228,10 +1228,10 @@ static bool reg_set_and_check(struct igb_adapter *adapter, u64 *data,
    "set/check reg %04X test failed: got 0x%08X expected 0x%08X\n",
    reg, (val & mask), (write & mask));
   *data = reg;
-  return 1;
+  return true;
  }
 
- return 0;
+ return false;
 }
 
 #define REG_PATTERN_TEST(reg, mask, write) \
-- 
1.7.1